### PR TITLE
Release/0.4.0

### PR DESCRIPTION
**Pull request by matus-tomlein**
## Changelog

* Add back-off on 100% event sending failure (#9)
* Add event-level Subject (#61)
* Add IP address property to Subject (#62)
* Rename send_limit to batch_size in Emitter (#63)

## Documentation

Documentation updates are in [this PR on data-value-resources](https://github.com/snowplow-incubator/data-value-resources/pull/102).

### CODE REVIEW

**Comment by snowplowcla** [comment]
Thanks for your pull request. Is this your first contribution to a Snowplow open source project? Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).

:memo: Please visit https://docs.snowplowanalytics.com/docs/contributing/contributor-license-agreement/ to learn more and sign.

Once you've signed, please reply here (e.g. I signed it!) and we'll verify. Thanks.

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,10 @@
+Version 0.3.0 (2022-04-27)
```
Comment (AlexBenny) [inline CHANGELOG]:
```suggestion
Version 0.4.0 (2022-04-27)
```

**Review by AlexBenny — APPROVED** [review]
👍 🚢   Just a small mistake in the Changelog

File: CHANGELOG (hunk 1 above)
Comment (matus-tomlein) [reply CHANGELOG]: Good catch, thanks!

**Pull request merged**